static/css/devhub/new-landing/sections/connect.less

```diff
@@ -0,0 +1,150 @@
+.DevHub-Connect{
+    h2 {
+        font-size: 24px;
+        @media @large {
+          font-size: 36px;
+        }
+    }
+    font-size: 14px;
+}
+
+.DevHub-content-container--Connect {
+    flex-direction: column;
+}
+
+.DevHub-content-header {
+    width: 100%
+}
+
+.Devhub-content-copy--Connect {
+    p, li {
+        color: @color-scheme-dark-items;
+    }
+    ul {
+        list-style-type: none;
+        padding: 0;
+    }
+    li {
+        line-height: 24px;
+        a {
+            .link(@color-scheme-dark-text);
+        }
+        margin-bottom: 20px;
+    }
+    ul.DevHub-content-copy--Connect-twitter-list li a {
+        display: block;
+    }
+
+    @media @medium {
+        display: flex;
+        max-width: 100%
+    }
+}
+
+.DevHub-Connect-section {
+    h4 {
+        margin: 10px 0;
+    }
+
+    @media @medium {
+        width: 33%;
+    }
+}
+
+.DevHub-Connect-newsletter-form {
+    display: flex;
+    flex-flow: row wrap;
+
+    label {
+        color: @color-scheme-dark-items;
+        font-size: 12px;
+    }
+
+    .form_group {
+        display: block;
+        margin-bottom: 20px;
+    }
+
+    .form_group-agree{
+        padding-left: 1.5em; // roughly width of a checkbox
```

Comment:
Shouldn't we support RTL too?

Reply:
yeah.  tbh I copied some of this css from the basket client example and extensionworkshop.

Reply:
https://github.com/mozilla/addons-server/pull/12669/commits/03d77ed1830768183dfdf6e3673f02bced7167cc

Reply:
https://github.com/mozilla/addons-server/pull/12669/commits/03d77ed1830768183dfdf6e3673f02bced7167cc